pkg/swarm/swarm.go

```diff
@@ -119,6 +119,15 @@ func (a Address) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a.String())
 }
 
+// Closer returns if x is closer to a than y
+func (x Address) Closer(a Address, y Address) bool {
+	cmp, err := DistanceCmp(a.b, x.b, y.b)
+	if err != nil {
+		return false
```

Comment:
i wouldn't swallow the error here. it assumes that x is not closer to y and that is not the correct assumption IMO. it would lead to flow control errors. better to propagate